badge.css

```diff
@@ -0,0 +1,164 @@
+
+body{
+    position: relative;
+    margin-left: 4px;
+    width: 100%;
+}
+.container{
+    /* background-color:skyblue ; */
+    left: 0;
+    padding: 3px;
+    display: flex;
+    gap: 5px;
+    position: relative;
+    /* z-index: 1; */
+}
+
+header{
+    display: flex;
+    justify-content: space-between;
+    margin: 5px;
+    background-color: rgb(155, 173, 238);
+    border-bottom-left-radius: 1rem;
+    border-bottom-right-radius: 1rem;
+    padding:.3rem;
+    top: 0;
+}
+
+.icon{
+    font-size: 2rem;
+    cursor: pointer;
+}
+
+.side-nav{
+    text-decoration: none;
+}
+
+.side-nav:hover{
+    background-color: turquoise;
+    padding: 5px 30px 5px 30px;
+    border-radius: 15px;
+}
+
+.sidebar{
+    width: 20rem;
+    height: fit-content;
+    /* border: 1px solid black; */
+    border-right-style: solid;
+    position: static;
+    text-align: center;
+}
+
+#header{
+    text-align: center;
+}
+
+.online,.offline{
+    width: 9rem;
+    height: 9rem;
+    border-radius: 50%;
+}
+
+.status{
+    position: relative;
+}
+
+.on{
+    position: absolute;
+    background-color: limegreen;
+    padding: 2px;
+    border-radius: 50%;
+    bottom: 0;
+    margin-left: -3rem;
+
+}
+
+.off{
+    position: absolute;
+    background-color: red;
+    padding: 2px;
+    border-radius: 50%;
+    bottom: 0;
+    margin-left: -3rem;
```

Comment:
i guess margin-left is not necessary you can use left instead depends on your use-case too